setup.py

```diff
@@ -62,6 +72,18 @@
 
 packages = find_packages(exclude=['docs', 'tests'])
 
+
+class LinkSpacyLanguageModel(install):
+    """Customized setuptools install command - prints a friendly greeting."""
+    def run(self):
+        from spacy.cli.link import link
+        try:
+            link('en_core_web_sm', 'en')
+        except:
+            print('Spacy Language Model already linked correctly.')
+        install.run(self)
```

Comment:
if you can run with the ugly name, it might be worth it instead of doing complex things at install time.